iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothub/FileUploadAndroidRunner.java

```diff
@@ -46,17 +44,4 @@ public static Collection inputsCommons() throws Exception
         isBasicTierHub = Boolean.parseBoolean(BuildConfig.IsBasicTierHub);
         return FileUploadTests.inputs();
     }
-
-    @Ignore
-    @Override
-    @Test
-    public void uploadToBlobAsyncSingleFileZeroLength()
```

Comment:
Q - why was this here?

Reply:
This test was failing on android in particular at some point, but it was never re-enabled

Reply:
are we keeping this now?

Reply:
File upload as a whole wasn't being tested on Android, but not on purpose. I'd re-enable it but there are some test failures on Android. They appear to be test bugs, not bugs in file upload itself. I'll open a separate PR to fix that bug, and to re-enable file upload tests on android